adding clarity to finch connect doc (#24)
Finch-API/docs: implementation-guide/Connect/Set-Up-Finch-Connect.mdx

Finch provides two options to setup Finch Connect: [Redirect](#redirect-finch-connect) or [Embedded](#embedded-finch-connect).

> ## Implementing Finch Connect to your application

Reviewer: ```suggestion
## Implementing Finch Connect in your application
```

      - `<script src="https://prod-cdn.tryfinch.com/v1/connect.js"></script>`
        > Since Finch Connect is an iFrame that requires interactivity, the HTML page that is loading Finch Connect **must be served from a server**. If the page is hosted statically, Finch Connect will not work properly.
    - [Redirect Flow](#redirect-to-finch-connect): The redirect flow is helpful in instances where you do not have a user interface (such as a link in an email) or you prefer to redirect in order to not host the authorization experience yourself.

2. **Configure Finch Connect**: Every flow requires you to configure Finch Connect with your unique `client_id`. You can find this `client_id` in your [Finch Developer Dashboard](https://dashboard.tryfinch.com/login).

## Redirect to Finch Connect

In this method of integrating Finch Connect, your application redirects your user's browser to Finch Connect hosted by Finch on `https://connect.tryfinch.com`. After a successful connection, Finch Connect will redirect your user back to a URI you specified (`redirect_uri`) with a short-lived authorization `code`.

> Additionally, set the `redirect_uri` parameter to the URL where users will be redirected after completing the authentication flow. The `redirect_uri` must be set in the [Finch Developer Dashboard](https://dashboard.tryfinch.com/login). Otherwise, the request will fail.

Reviewer: ```suggestion
Set the `redirect_uri` parameter to the URL where users will be redirected after completing the authentication flow. The `redirect_uri` must be set in the [Finch Developer Dashboard](https://dashboard.tryfinch.com/login). Otherwise, the request will fail.
```
i dont think we need "Additionally" given the the line above

## Embedded Finch Connect

> If you are using a [Finch-provided SDK](/developer-resources/SDKs#frontend-sdks) (recommended), you are essentially embedding the Finch Connect screen into your application. The user will remain entirely on your application throughout the entire process. When the `onSuccess` event is called by the SDK, simply pass the `code` to your internal callback endpoint as a query parameter. 

Reviewer: ```suggestion
The [Finch-provided SDK](/developer-resources/SDKs#frontend-sdks) embeds the Finch Connect screen into your application. The user will remain entirely on your application throughout the process. When the `onSuccess` event is called by the SDK, simply pass the `code` to your internal callback endpoint as a query parameter. 
```
just revising to be a little less wordy

> > NOTE: You will not include a `redirect_uri` in an embedded flow because the entire flow is already self-contained in your app; no redirect necessary.

Reviewer: ```suggestion
> NOTE: You should not include a `redirect_uri` if using the embedded flow. Because the entire flow is already self-contained in your app, no redirect is necessary.
```
